mpp/include/mpp_util.inc

```diff
@@ -1389,8 +1398,8 @@ end function rarray_to_char
                    call mpp_error(FATAL, 'get_ascii_file_num_lines: Error reading line '//trim(text)// &
                         ' in file '//trim(FILENAME)//'.')
                 end if
-                if ( len_trim(str_tmp) == LENGTH) then
-                   write(UNIT=text, FMT='(I5)') LENGTH
+                if ( len_trim(str_tmp) == LENGTH ) then
+                   write(UNIT=text, FMT='(I5)') length
```

Comment:
I know that Fortran does not enforce alphanumeric case, but we should leave this as `LENGTH`